fix: UTC-257: Blueprint url is undefined when accessing from projects table mode

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17589212166)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/573
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17587014430

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17589130678)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17589130678

mcanu committed:
Merge branch 'develop' into 'fb-utc-257'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17589130678

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17589212166

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17590271958
